tests/e2e/integration/networks.js

```diff
@@ -14,102 +14,94 @@ describe('Test cases for networks page', () => {
+      .click()
+      .enterNetworkDetails(
+        defaultNetwork.name,
+        defaultNetwork.url,
+        defaultNetwork.middlewareUrl,
+        defaultNetwork.compilerUrl,
+      )
+      .get('[data-cy=connect]')
+      .buttonShouldBeDisabled('[data-cy=connect]')
+      .get('[data-cy=input-message]')
+      .should('exist')
+
+      .enterNetworkDetails('test', 'test', 'test', 'test')
+      .get('[data-cy=connect]')
+      .buttonShouldBeDisabled('[data-cy=connect]')
+      .get('[data-cy=input-message]')
+      .should('exist')
+
+      .get('[data-cy=cancel]')
+      .should('be.visible')
+      .click()
+      .urlEquals('/more/settings/networks');
   });
 
-  // it('Can add, select, edit, delete new network, can not add network with the same name', () => {
-  //   cy.addNetwork(
-  //     'Newnetwork',
-  //     defaultNetwork.url,
-  //     defaultNetwork.middlewareUrl,
-  //     defaultNetwork.compilerUrl,
-  //   )
-  //     .get('[data-cy=close]')
-  //     .click()
-  //     .openNetworks()
-  //     .get('[data-cy=more]')
-  //     .click()
-  //     .get('[data-cy=edit]')
-  //     .should('be.visible')
-  //     .click()
-  //     .get('[data-cy=network] input')
-  //     .should('have.value', 'Newnetwork')
-  //     .get('[data-cy=url] input')
-  //     .should('have.value', defaultNetwork.url)
-  //     .clear()
-  //     .type(defaultNetwork.url)
-  //     .get('[data-cy=middleware] input ')
-  //     .should('have.value', defaultNetwork.middlewareUrl)
-  //     .clear()
-  //     .type(defaultNetwork.middlewareUrl)
-  //     .get('[data-cy=connect]')
-  //     .click()
-  //     .get('[data-cy=network-url]')
-  //     .eq(-1)
-  //     .should('contain', defaultNetwork.url)
-  //     .get('[data-cy=network-middleware]')
-  //     .eq(-1)
-  //     .should('contain', defaultNetwork.middlewareUrl)
-  //     .goBack()
-  //     .goBack()
-  //     .goBack()
-  //     // TODO should be aligned with the new home page changes
-  //     // .get('.transaction-list', { timeout: 8000 })
-  //     // .should('exist')
-  //
-  //     .openNetworks()
-  //     .get('[data-cy=to-add]')
-  //     .click()
-  //     .enterNetworkDetails(
-  //       'Newnetwork',
-  //       defaultNetwork.url,
-  //       defaultNetwork.middlewareUrl,
-  //       defaultNetwork.compilerUrl,
-  //     )
-  //     .get('[data-cy=connect]')
-  //     .click()
-  //     .buttonShouldBeDisabled('[data-cy=connect]')
-  //     .get('[data-cy=cancel]')
-  //     .click()
-  //
-  //     .get('[data-cy=more]')
-  //     .click()
-  //     .get('[data-cy=delete]')
-  //     .should('be.visible')
-  //     .click()
-  //     .get('[data-cy=networks] .network-row')
-  //     .should('have.length', 2)
-  //     .get('[data-cy=networks] .network-row')
-  //     .eq(1)
-  //     .find('.checkmark')
-  //     .should('have.class', 'checked');
-  // });
+  it('Can add, select, edit, delete new network, can not add network with the same name', () => {
```

Comment:
I hate the idea of having one long test that covers many steps. Especially with this kind of "go backs":
```
      .goBack()
      .goBack()
      .goBack()
      .goBack()
      .goBack()
      .goBack()
      .goBack()
```
Maybe it would be good to split this into separate "add", "select", "edit", "delete" ?